app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,32 @@
+class SessionsController < ApplicationController
+    before_action :check, only: [:new]
+
+    def new
+    end
```

Comment:
下記のように1行での記載が可能です。これにより、中身を書き忘れていないことを明示できます。
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyMethod
```suggestion
    def new; end
```